app/src/parser/transaction.rs

```diff
@@ -198,6 +186,62 @@ impl<'a> Transaction<'a> {
         Ok(fields)
     }
 
+    fn format_output(
```

Comment:
Maybe some rust tests for this too?

Reply:
the unit tests in the int_format module are sufficient as it is what that function uses